packages/bazel-bot/cloudbuild.yaml

```diff
@@ -13,4 +13,7 @@
 # limitations under the License.
 
 steps:
-  - name: 'gcr.io/repo-automation-bots/bazel-bot'
\ No newline at end of file
+  - name: 'gcr.io/repo-automation-bots/bazel-bot'
+    env:
+      - 'BAZEL_FLAGS=--google_default_credentials --remote_cache=https://storage.googleapis.com/repo-automation-bots-bazel-cache'

```

Comment:
Is this refactor related? or just cleanup.

Reply:
An unrelated improvement.